From c22c4e5c238d7864ef6e374f73fde962487c9869 Mon Sep 17 00:00:00 2001
From: Jonas Karlman <jonas@kwiboo.se>
Date: Mon, 27 Apr 2026 01:26:24 +0000
Subject: [PATCH 194/247] WIP: drm/bridge: dw-hdmi: attach Broadcast RGB
 property

Attach the Broadcast RGB connector property to the dw-hdmi bridge
connector.

dw-hdmi already reads conn_state->hdmi.is_limited_range and can use the
CSC block to convert RGB full range to limited range. Expose the
standard connector property so userspace can request that behavior.

Signed-off-by: Jonas Karlman <jonas@kwiboo.se>
---
 drivers/gpu/drm/bridge/synopsys/dw-hdmi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
index 487830f55666..6e54a307fda0 100644
--- a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
+++ b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
@@ -2349,7 +2349,7 @@ static int dw_hdmi_connector_create(struct dw_hdmi *hdmi)
 	if (IS_ERR(connector))
 		return PTR_ERR(connector);
 
-	return 0;
+	return drm_connector_attach_broadcast_rgb_property(connector);
 }
 
 /* -----------------------------------------------------------------------------
-- 
2.43.0

